routers/api/actions/runner/utils.go

```diff
@@ -113,7 +114,7 @@ func generateTaskContext(t *actions_model.ActionTask) *structpb.Struct {
 		"base_ref":          baseRef,                                              // string, The base_ref or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is either pull_request or pull_request_target.
 		"env":               "",                                                   // string, Path on the runner to the file that sets environment variables from workflow commands. This file is unique to the current step and is a different file for each step in a job. For more information, see "Workflow commands for GitHub Actions."
 		"event":             event,                                                // object, The full event webhook payload. You can access individual properties of the event using this context. This object is identical to the webhook payload of the event that triggered the workflow run, and is different for each event. The webhooks for each GitHub Actions event is linked in "Events that trigger workflows." For example, for a workflow run triggered by the push event, this object contains the contents of the push webhook payload.
-		"event_name":        t.Job.Run.Event.Event(),                              // string, The name of the event that triggered the workflow run.
```

Comment:
The old data in db could have empty `TriggerEvent`, maybe we can fallback to `Event.Event()` if it does.